api/serializers.py

```diff
@@ -27,3 +27,37 @@ class ExportInstructorLocationsSerializer(serializers.ModelSerializer):
     class Meta:
         model = Airport
         fields = ('name', 'latitude', 'longitude', 'instructors')
+
+
+class EventSerializer(serializers.ModelSerializer):
+    humandate = serializers.SerializerMethodField()
+    country = serializers.CharField()
+
+    def get_humandate(self, obj):
+        """Render start and end dates as human-readable short date."""
+        return EventSerializer.human_readable_date(obj.start, obj.end)
+
+    @staticmethod
+    def human_readable_date(date1, date2):
+        """Render start and end dates as human-readable short date."""
+        if date1 and not date2:
+            return '{:%b %d, %Y}-???'.format(date1)
+        elif date2 and not date1:
+            return '???-{:%b %d, %Y}'.format(date2)
+        elif not date2 and not date1:
+            return '???-???'
+
+        if date1.year == date2.year:
+            if date1.month == date2.month:
+                return '{:%b %d}-{:%d, %Y}'.format(date1, date2)
+            else:
+                return '{:%b %d}-{:%b %d, %Y}'.format(date1, date2)
+        else:
+            return '{:%b %d, %Y}-{:%b %d, %Y}'.format(date1, date2)
```

Comment:
There would be two ways to do it and both have flaws:

1. Looking up humandate from index.html file of each workshop means a lot of requests to GitHub - something we wanted to cut out.
2. Storing humandate within AMY database: humandate does not follow one formatting + we can easily calculate it, so why store it?

So I'd not store humandate anywhere.